docs/fidesops/docs/guides/query_execution.md

```diff
@@ -2,17 +2,17 @@
 
 ## Graphs and Traversals
 
-Fidesops uses the datasets you provide to generate a _graph_ of the resources. Based on the identity data you provide, Fidesops then generates a specific _traversal_, which is the order of steps that will be taken to fulfill the specific request. If no traversal can be generated that reaches all declared collections, Fidesops will generate an error.
+Fidesops uses your Datasets to generate a _graph_ of the resources. Based on the identity data you provide, Fidesops then generates a specific _traversal_, which is the order of steps that will be taken to fulfill a specific request. 
 
-The graph supports both directed and non-directed edges using the optional `direction` parameter on the relation (non-directional edges may be traversed in either direction). You can always preview the queries that will be generated or manually control the order of operations by making relations explicitly directional and with the `after` dataset and collection parameters. 
+The graph supports both directed and non-directed edges using the optional `direction` parameter on the relation (non-directional edges may be traversed in either direction). You can preview the queries that will be generated or manually control the order of operations by making relations explicitly directional and with the  and Collection parameters. If you specify a Collection that can't be reached, Fidesops generates an error.
```

Comment:
There's an erroneous double space on this line between `the  and Collection parameters.`